### DISCUSSION TITLE
I've never seen an `etc` directory in `/usr` directory - is this something new?

### DISCUSSION THREAD

**Discussion by ak2766** [acassen/keepalived#2462] [General]
Is there new consensus in Linux community on using `/usr/etc` as a configuration location as opposed to either `/usr/local/etc` or `/etc/opt/`?

I don't see mention of `/usr/etc` at [FHS](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard)!

**Comment by ak2766** [comment]
I'm asking due to this message in the logs:
```
Keepalived[555]: WARNING - using deprecated default config file '/etc/keepalived/keepalived.conf' - please move to '/usr/etc/keepalived/keepalived.conf'
```

**Reply by pqarmitage** [reply]
The paths are generated by the configure script. I can generate the above message if I specify `--prefix=/usr --with-default-config-file=/usr/etc/keepalived/keepalived.conf` as options to configure.
